### DISCUSSION TITLE
Get the camera frame ID starting from zero

### DISCUSSION THREAD

**Discussion by lachioma** [bonsai-rx/bonsai#1641] [Q&A] [answered]
I have a workflow to acquire images from a FLIR camera. I would like to get a frame ID that starts at zero (or one) at each acquisition. However, the frame ID that you get from the camera via `ChunkData.FrameID` does not start from zero or one in each acquisition, but its value is always incremental across multiple separate acquisitions..

I would thus like to subtract each frame ID by the first frame ID. 
I tried with the following but it's not quite right. What is the best way to achieve this?

Thank you for any feedback!

![image](https://github.com/bonsai-rx/bonsai/assets/29898879/2230238e-344f-4544-b68f-679f2c4a1ebf)

**Comment by bruno-f-cruz** [comment, marked as answer]
Probably something like this:
![image](https://github.com/bonsai-rx/bonsai/assets/7049351/1ba1b986-05cb-47a9-95eb-d0be78086146)
